db-migration.Dockerfile

```diff
@@ -1,4 +1,15 @@
-FROM liquibase/liquibase:3.10.x
+# Use Liquibase as base image
+FROM liquibase/liquibase:4.25
 
-COPY --chown=liquibase:liquibase changelog ./changelog
-COPY --chown=liquibase:liquibase scripts /scripts
\ No newline at end of file
+# Change to root user for installation
+USER root
```

Comment:
Won't AKS policy block this running with elevated user access?
@ppotturi  @jeevankuduvaravindran  @ajaysoni-cap

Reply:
@kbabigumira , As discussed, we can have a look at this during policy implementation